doc/Android.md

```diff
@@ -0,0 +1,109 @@
+**Document based on Minetest `5.6.1` Android build**
+# Minetest: Android version
+Minetest for Android are the same Minetest on Windows but compiled for Android, not seperate version
+## Controls
+The Android port doesn't support everything you can do on PC due to the
+limited capabilities of common devices. What can be done is described
+below:
+
+While you're playing the game normally (that is, no menu or inventory is
+shown), the following controls are available:
+* Look around: touch screen and slide finger
+* double tap: place a node
+* long tap: dig node or use the holding item
+* touch buttons: press button
+* Buttons:
+
+1. left upper corner: chat
+2. right lower corner: jump
+3. right lower corner: crouch
+4. left lower corner (Joystick): walk/step...
+5. left lower corner: display inventory
```

Comment:
You could inline the actual images here (see textures/base/pack) for better understanding.